tests/integration/IncludesForTests.php

```diff
@@ -2,6 +2,11 @@
 
 declare( strict_types=1 );
 
+/**
+ * @todo add this to the bootstrap
+ * Transform into a listener
```

Comment:
Can you explain the intention here?

Reply:
This was an internal note when I was implementing the PHPUnit configuration.
PHPUnit has some functionalities that I believe could be helpful for this project, as listeners, bootstrap, constant configuration. 

Things that I wanted to propose were:
- Adding this "class"'s functions into a listener so that they could be configured in the phpunit.xml; 
- Adding .env for config vars/bootstrap.php to define env vars
- Adding the constants to the phpunit.xml
- Adding PHPUnit into a composer and 
But this should be for the future, and I thought I was moving too many things simultaneously. I'm removing this comment for now since I don't have it organized atm.